### ISSUE TITLE
Savegame notification

### ISSUE THREAD

**Issue by benhandschuh** [jsettlers/settlers-remake#613]
Show a notification when game is saved after pressing F2.

**Comment by Tom-Pratt** [comment]
This is a good idea. The android version already shows a Toast when you save the game.

However, it shows the toast when you click the save button, not when the game has been correctly saved. (Would be better if the game could say that the save has completed successfully)

I have noticed a bug where if you save the game and then quit  quickly, you cannot resume the game properly. (Settlers fly around in straight lines, probably in the direction they were going when you.clicked save)

This is probably more common on android where computation is.much slower than on a PC. PC saves probably happen almost instantly but on slower android devices  this doesn't seem to be the case

**Comment by Tom-Pratt** [comment]
The game should probably freeze until the save has successfully completed

**Comment by andreas-eberle** [comment]
I agree, we will need to provide a callback mechanism to tell the UI that saving the game has been finished successfully.

**Comment by michaelzangl** [comment]
You can use a normal message for this, like the attacked message. This will then show up on both Android and Swing.
